src/reducers/breakpoints.js

```diff
@@ -185,24 +184,19 @@ export function getBreakpointForLine(
   return breakpoints.find(breakpoint => breakpoint.location.line === line);
 }
 
-export const getHiddenBreakpoint = createSelector(getBreakpoints, function(
-  breakpoints
-) {
-  const hiddenBreakpoints = breakpoints
+export function getHiddenBreakpoint(state: OuterState) {
+  return getBreakpoints(state)
     .valueSeq()
     .filter(breakpoint => breakpoint.hidden)
```

Comment:
maybe we could use `find`. it will return the first match, though it doesnt need to be done in this pr.. this also looks like it isnt part of the pause refactor. I bring this up mostly so that we have a clear chronology when looking back on the commit messages. don't worry about it for this one, but it might be worth splitting out these driveby refactors into their own prs

Reply:
yeah... i did it because `createSelector` was no longer defined... but that might be more troubling :/